SwiftyBeaverTests/BaseDestinationTests.swift

```diff
@@ -343,9 +343,14 @@ class BaseDestinationTests: XCTestCase {
             path: "/world/ViewController.swift",
             function: "myFunc",
             message: "Hello World"))
+        
+        XCTAssertFalse(destination.shouldLevelBeLogged(.Verbose,
+            path: "/world/ViewController.swift",
+            function: "myFunc",
+            message: "Hello World"))
 
         // not in filter but matching global minLevel
-        XCTAssertTrue(destination.shouldLevelBeLogged(.Info,
+        XCTAssertFalse(destination.shouldLevelBeLogged(.Info,
```

Comment:
I am looking at filters not to filter logs that don't meet the minLevel but a way to change what logs I see. When I see filters that's what I think. Sounds like you should have a flag or something that says filter only non minLevel logs or filter all available logs.

I have a lot of logs in my code, but if i am working on one area I just want to filter those logs and not see all the other logs. So I usually put a contains for one or two classes I am working on. But I need the excludes for larger classes where I where I filter that one class but want to exclude a couple methods.